Adds more mob crates for livestock

Peptide90 opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Adds critter crate sprite as well as crates for NPC mob crates for all current animals.
Adds crates for:
**bee, butterfly, cat, chicken, corgi, cow, goat, goose, gorilla, monkeycube, parrot, penguin, snake**

**Screenshots**
<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->
![livestockcrate](https://user-images.githubusercontent.com/78795277/110504121-c1661380-80f4-11eb-9ace-966395360b26.png)
The crate sprite being used. Is from Para but is the same as the TG one (just the door meta is different so used para as fits ours. Cev was a green shipping container so can change but looked disproportional...comments?).
**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.
-->
🆑

- add: Adds Animal / critter mob crates to cargo

Peptide90 commented:
> 
> 
> Please write a proper changelog

As you wrote that I edited. Will finish this PR tomorrow, ran out of time today.

Zumorica commented:
> > Please write a proper changelog
> 
> As you wrote that I edited. Will finish this PR tomorrow, ran out of time today.

No worries, take your time!

SweptWasTaken commented:
Please format RSI correctly

Peptide90 commented:
> 
> 
> Please format RSI correctly

- Corrected RSI issue in latest commit. 
- All tested in game and working at cargo.
- Reduced butterfly mob from 10 to 5 so that it's not so griefable.
- All crates currently cost 1000 points which can be rebalanced in future by whoever should penguin and goose provide for rarer cooking ingredients.
- Added construction graph for the wooden crate (crafting) but currently deconstruct doesn't work so this will need to be looked into. (Historically, crates / lockers needed welded before deconstruction).
- Changed physics mass for mob crates so that they're heavier and also added destructible components so they can be destroyed open.

PaulRitter reviewed (changes requested):
please fix conflicts

Peptide90 commented:
> 
> 
> please fix conflicts

Fixed...damn mouse mobs...

SweptWasTaken commented on Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml:
```diff
@@ -40,6 +40,19 @@
       - cargo.funplushies
       - cargo.funartsupplies
       - cargo.funinstruments
+      - livestockbee
```
Format these IDs correctly.

Peptide90 replied:
This is how @Zumorica now wants them. Gone with the cargo prefix. Was going to do the rest at some point.

SweptWasTaken commented:
Content part of this PR is good. I disagree with adding them to the cargo console. Remove them and it's mergeable.

SweptWasTaken commented:
Some maintainer can probably fix conflicts really easily for this and then merge it.

Peptide90 commented:
> Some maintainer can probably fix conflicts really easily for this and then merge it.

Will fix Friday unless someone beats me to it.
